ReactWindows/ReactNative/Modules/DeviceInfo/DeviceInfoModule.cs

```diff
@@ -183,29 +174,63 @@ private void OnOrientationChanged(DisplayInformation displayInformation, object
         private void SendUpdateDimensionsEvent()
         {
             Context.GetJavaScriptModule<RCTDeviceEventEmitter>()
-                .emit("didUpdateDimensions", GetDimensions());
+                .emit("didUpdateDimensions", GetAllDimensions());
```

Comment:
Do we always have to send all dimensions?

Reply:
Yes we have, as RN.Dimensions keeps the last dimensions only